stdlib/3/collections/__init__.pyi

```diff
@@ -100,28 +108,28 @@ class deque(MutableSequence[_T], Generic[_T]):
     def __delitem__(self, s: slice) -> None: raise TypeError
 
     def __contains__(self, o: object) -> bool: ...
+    def __reversed__(self) -> Iterator[_T]: ...
 
-    # TODO __reversed__
+    if sys.version_info >= (3, 5):
+        def __add__(self, other: deque[_T]) -> deque[_T]: ...
+        def __mul__(self, other: int) -> deque[_T]: ...
+        def __imul__(self, other: int) -> None: ...
 
 
 class Counter(Dict[_T, int], Generic[_T]):
     @overload
-    def __init__(self) -> None: ...
+    def __init__(self, **kwargs: int) -> None: ...
     @overload
-    def __init__(self, Mapping: Mapping[_T, int]) -> None: ...
+    def __init__(self, mapping: Mapping[_T, int]) -> None: ...
     @overload
     def __init__(self, iterable: Iterable[_T]) -> None: ...
-    # TODO keyword arguments
 
     def elements(self) -> Iterator[_T]: ...
 
-    @overload
-    def most_common(self) -> List[_T]: ...
-    @overload
-    def most_common(self, n: int) -> List[_T]: ...
+    def most_common(self, n: int = ...) -> List[_T]: ...
 
     @overload
-    def subtract(self, Mapping: Mapping[_T, int]) -> None: ...
+    def subtract(self, mapping: Mapping[_T, int]) -> None: ...
```

Comment:
Strictly speaking this should probably be `__mapping`, since it's a positional-only argument.